.jest/configs/tables-function-async/jest-dynalite-config.js

```diff
@@ -0,0 +1,20 @@
+const realTimeout = setTimeout;
```

Comment:
Why rename an STL function?

Reply:
The test suite enables fake timers, so when jest-dynalite goes to execute the tables function the timeout has been mocked. The reference is just to allow the "sleep" function to work after the timers have been faked.

Reply:
I see